File: Programming Basics with JS/T05 - For Loop/exercise/05. Salary.js
function salary(input) {
    let tabs = Number(input[0]);
    let salary = Number(input[1]);

    for (let i = 2; i <= tabs + 1; i++) {
        if (input[i] === 'Facebook') {
            salary -= 150;
        } else if (input[i] === 'Instagram') {
            salary -= 100;
        } else if (input[i] === 'Reddit') {
            salary -= 50;
        }

        if (salary <= 0) {
            console.log(`You have lost your salary.`);
            return;
        }
    }
    console.log(parseInt(salary));
}
salary(['10', '750', 'Facebook', 'Dev.bg', 'Instagram', 'Facebook', 'Reddit', 'Facebook', 'Facebook']);
salary(['3', '500', 'Github', 'Stackoverflow', 'SoftUni']);
salary(['3', '500', 'Facebook', 'Stackoverflow', 'SoftUni']);

File: Programming Basics with Python/T05 - For Loop/exercise/05. Salary.py
number_of_tabs = int(input())
salary = int(input())

for _ in range(number_of_tabs):
    current_tab_name = str(input())

    if current_tab_name == "Facebook":
        salary -= 150
    elif current_tab_name == "Instagram":
        salary -= 100
    elif current_tab_name == "Reddit":
        salary -= 50
    if salary <= 0:
        print("You have lost your salary.")
        break

if salary > 0:
    print(f"{salary}")